frontend/javascripts/oxalis/model/sagas/mapping_saga.ts

```diff
@@ -119,23 +167,16 @@ function* maybeFetchMapping(
 
   const dataset = yield* select((state) => state.dataset);
   const layerInfo = getLayerByName(dataset, layerName);
-  const params = [
-    dataset.dataStore.url,
-    dataset, // If there is a fallbackLayer, request mappings for that instead of the tracing segmentation layer
-    "fallbackLayer" in layerInfo && layerInfo.fallbackLayer != null
-      ? layerInfo.fallbackLayer
-      : layerInfo.name,
-  ] as const;
-  const [jsonMappings, serverHdf5Mappings] = yield* all([
-    call(getMappingsForDatasetLayer, ...params),
-    call(getAgglomeratesForDatasetLayer, ...params),
-  ]);
+
   // Make sure the available mappings are persisted in the store if they are not already
   const areServerHdf5MappingsInStore =
     "agglomerates" in layerInfo && layerInfo.agglomerates != null;
-  if (!areServerHdf5MappingsInStore) {
-    yield* put(setLayerMappingsAction(layerName, jsonMappings, serverHdf5Mappings));
-  }
```

Comment:
@daniel-wer In this commit, I changed the logic slightly, but I'm not too sure about this code. Maybe you remember this bit better? Two things:

- the old code always fetched the mappings/agglomerates from the server (even when they already existed). I changed that so that the store is used if they already exist
- the loaded agglomerates were only updated in the store if no agglomerates existed before. I somehow kept that, but I don't think that it really makes sense, because loadLayerMappings could simply do the update if it loaded something from the server (no extra parameter required). but I wanted to ask you about this logic before I'm butchering it any further :sweat_smile: 

might make sense to have a call about it, if you prefer that :call_me_hand: